commons/commons-serialization/src/main/kotlin/ac/uk/ebi/biostd/tsv/deserialization/TsvSerializationContext.kt

```diff
@@ -48,13 +49,17 @@ class TsvSerializationContext {
         execute(rootSection, chunk) { rootSection.addSectionTable(chunk.asTable()) }
 
     fun addSubSectionTable(parent: String, chunk: SectionTableChunk) =
-        execute(rootSection, chunk) { sections.getValue(parent).addSectionTable(chunk.asTable()) }
+        execute(rootSection, chunk) {
+            sections.getOrElse(parent, { throw InvalidParentSectionException(parent) }).addSectionTable(chunk.asTable())
```

Comment:
I think we should keep this part clean and instead create specialized Map that handles this scenario something like 

```
SectionsContext {
    getSection(parentCccNo:String)
}
```